File: Medii de Proiectare si Programare/TrackAndField-C#/TrackAndField/model/Participant.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TrackAndField.model
{
    class Participant : Entity<string>
    {
        public string Name { get; set; }
        public int Age { get; set; }

        public Participant(string name, int age)
        {
            Name = name;
            Age = age;
        }
        public override string ToString()
        {
            return Name + " | " + Age;
        }
    }
}

File: Medii de Proiectare si Programare/TrackAndField-Java/src/main/java/model/Participant.java
package model;

public class Participant extends Entity<String> implements Comparable {
    private String name;
    private int age;

    public Participant(String name, int age) {
        this.setId(name);
        this.name = name;
        this.age = age;
    }

    public String getName() { return name; }

    public void setName(String name) {
        this.name = name;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    @Override
    public int compareTo(Object o) {
        Participant other = (Participant)o;
        return Integer.compare(this.age, other.age);
    }

    @Override
    public String toString() {
        return this.name + " | " + this.age;
    }
}
